hubtraf/check.py

```diff
@@ -44,10 +46,15 @@ def main():
         'username',
         help='Name of user to check'
     )
+    argparser.add_argument(
```

Comment:
I don't think we need this - usually when you are using env vars for config, their names aren't configurable.

Instead, I think we should get the variable from `os.environ` here, and pass the actual token through to `check_user`, rather than having it call `os.environ` by itself. This also makes testing easier - relying on `os.environ` is the same as relying on global state, and keeping that out of internals of code is a good idea.

Reply:
Done. Thanks @yuvipanda :heart: